scripts/.rpm-helpers

```diff
@@ -1,43 +1,43 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
 
-
-gen-test-ver() {
-    local version=$1
-    local suffix=$2
-    local base_release_ver=$3
-    SPEC_FILE=${SPEC_FILE:-SPECS/containerd.spec}
-    test_num=${version#*-${suffix}.}
-    without_rpm_ver="${base_release_ver}.${test_num}.${suffix}.${test_num}"
-    # This only really matters for blessed builds (i.e. builds that are only rpm specific or have a version specific in the spec)
-    # should output the latest rpm build number so that it gets placed accordingly in the repo
-    if grep -q "${version%-${suffix}.*}-${without_rpm_ver}" "${SPEC_FILE}"; then
-        grep -o "${version%-${suffix}.*}-${without_rpm_ver}.[0-9]*" "${SPEC_FILE}" | head -n 1 | cut -d"-" -f 2
-    else
-        echo "${without_rpm_ver}.1"
-    fi
+gen_test_ver() {
+	version=$1
```

Comment:
is there a linter that prefers global variables over local ?

Reply:
local is only supported on bash, so could fail when using in a Dockerfile

Reply:
I could use a sub-shell for this, but I don't think it was needed in this case

Reply:
^ updated the functions to use sub-shells so that setting environment variables in a function does not change the current environment